fetch_forecasts.py

```diff
@@ -59,13 +63,28 @@ def wget(self, flavor, yearmonth=None):
             path = os.path.join(self.root, flavor, yearmonth, '')
         else:
             path = os.path.join(self.root, flavor, '')
-        cmd += [f"https://{path}"]
+        if ftp:
+            insert = model_info.ftp_source.get(self.model).get(flavor)
+            cmd += ["ftp://" + insert]
+            cmd += ["-P", 'tmp']
+            cmd += ["--cut-dirs=1000", "-nH"]
+            cmd2 = ["source_dir='tmp'; dest_dir='" + model_info.model_root.get(self.model) + os.sep + flavor + "'; mkdir -p \"$dest_dir\"; find \"$source_dir\" -type f -name 'archive_[0-9][0-9][0-9][0-9][0-9][0-9]*.html' | while read -r file; do filename=$(basename \"$file\"); year_month=$(echo \"$filename\" | grep -oP 'archive_\\K[0-9]{6}'); year=$(echo $year_month | grep -o '^[0-9]\{4\}'); month=$(echo $year_month | grep -o '..$'); target_dir=\"$dest_dir/$year/$month\"; mkdir -p \"$target_dir\"; mv \"$file\" \"$target_dir/\"; done; rm -r \"$source_dir\";"]
```

Comment:
From chat with Luke:

All that command is doing is reading the filenames (archive_YYYYMMDD.html), making subdirectories (YYYY/MM), and putting the files in the correct place.
 
I am not a fan of this.